airship_tempest_plugin/services/shipyard/json/airflow_monitoring_client.py

```diff
@@ -30,10 +30,16 @@ def list_workflows(self):
         resp, body = self.get('workflows')
         self.expected_success(200, resp.status)
         body = json.loads(body)
-        return rest_client.ResponseBody(resp, body)
+        if isinstance(body, list):
+            return rest_client.ResponseBodyList(resp, body)
+        else:
+            return rest_client.ResponseBody(resp, body)
 
     def get_workflow(self, workflow_id=None):
         resp, body = self.get('workflows/%s' % workflow_id)
         self.expected_success(200, resp.status)
         body = json.loads(body)
-        return rest_client.ResponseBody(resp, body)
+        if isinstance(body, list):
```

Comment:
as this is a single GET would it ever return a list?

Reply:
Modified accordingly as it's returning a specific workflow